pkg/auth/auth_utils.go

```diff
@@ -34,3 +36,10 @@ func shouldStorePayloadInCache(err error) bool {
 	}
 	return false
 }
+
+func isUpdateRequest(request *http.Request) bool {
+	return request != nil && (request.Method == http.MethodPost ||
+		request.Method == http.MethodPut ||
+		request.Method == http.MethodPatch ||
+		request.Method == http.MethodDelete)
```

Comment:
maybe we should handle delete separately. i.e. we can check against AMS whether delete action is permitted instead of update. just in case the role model would change.

Reply:
I thought we agreed to have the update action for delete?

Reply:
fixed